### ISSUE TITLE
Different "server_version" result from /v2/info between official xenon node and Release 2.0.0-rc1

### ISSUE THREAD

**Issue by tyGavinZJU** [blockstack/stacks-blockchain#2299]
**Describe the bug**
When launching the stacks-node binary in https://github.com/blockstack/stacks-blockchain/releases/tag/2.0.0-rc1, it returns the result from /v2/info:

```
{"peer_version":385875968,"pox_consensus":"dd440316a2b1c9a28134c025abd50dfe4238386b","burn_block_height":1902561,"stable_pox_consensus":"efa4f45d2ae830cf5ee7cc864c120b68cdaf334b","stable_burn_block_height":1902554,"server_version":"stacks-node 0.0.1 (:+, release build, linux [x86_64])","network_id":2147483648,"parent_network_id":118034699,"stacks_tip_height":0,"stacks_tip":"0000000000000000000000000000000000000000000000000000000000000000","stacks_tip_consensus_hash":"0000000000000000000000000000000000000000","genesis_chainstate_hash":"1d587f628d370651ade417d6f92148210e1f1c367a8afd35c4af87781d61a2d9","unanchored_tip":"4ec7ae4a471502547f69ff7165ce5eab0257def1a6607770512fb66e5fcc7e79","unanchored_seq":0,"exit_at_block_height":null}
```

The ```server_version``` is ```"stacks-node 0.0.1 (:+, release build, linux [x86_64])"```
Which is different from the one running in http://xenon.blockstack.org:20443/v2/info

Which is ```"stacks-node 2.0.0-rc1 (master:444e71f+, release build, linux [x86_64])"```

I am not sure if it will influence the stacks-node.

**Steps To Reproduce**
Download the binary in https://github.com/blockstack/stacks-blockchain/releases/tag/2.0.0-rc1 and running 
```./stacks_node start --config=...toml```

And after start the spawn server, you can visit the /v2/info to see the info above.

I have tried in darwin and linux x64, it all shows like ```"server_version":"stacks-node 0.0.1 (:+, release build, ......)```

**Expected behavior**
Show commit info.

**Environment (please complete the following information):**
 - OS: [e.g. Ubuntu / Debian]
 - Rust version
 - Version of the appropriate binary / software package: https://github.com/blockstack/stacks-blockchain/releases/tag/2.0.0-rc1

**Comment by mike0182** [comment]
exactly the same problem
OS: Ubuntu 18.04
Rust version: rustc 1.49.0 (e1884a8e3 2020-12-29)
Version of the appropriate binary / software package: https://github.com/blockstack/stacks-blockchain/releases/tag/2.0.0-rc1

**Comment by diwakergupta** [comment]
> The `server_version` is `"stacks-node 0.0.1 (:+, release build, linux [x86_64])"`
> Which is different from the one running in http://xenon.blockstack.org:20443/v2/info
> 
> Which is `"stacks-node 2.0.0-rc1 (master:444e71f+, release build, linux [x86_64])"`
> 
> I am not sure if it will influence the stacks-node.

Thanks for the report @tyGavinZJU -- there's no functional impact, if you downloaded the pre-built binaries, or if you compiled the tag from source, you should be good to go! This is a minor issue due to some changes in our build process recently. Should be an easy fix, we'll address it later today.

**Comment by diwakergupta** [comment]
@CharlieC3 I think the fix for #2275 may have broken this, could you take a look?

**Comment by CharlieC3** [comment]
Seems #2275 only covered the fix for our Docker image, but not the binaries produced. I'll put out a fix for this, but like Diwaker said above the binaries _are_ correctly correlating to the release they're under when downloading them.

**Comment by tyGavinZJU** [comment]
> Seems #2275 only covered the fix for our Docker image, but not the binaries produced. I'll put out a fix for this, but like Diwaker said above the binaries _are_ correctly correlating to the release they're under when downloading them.

Will you update the binary in 2.0.0rc release? @CharlieC3

**Comment by CharlieC3** [comment]
@tyGavinZJU This fix will be included in the next release, whether it's `2.0.0-rc2` or `2.0.0`., whichever one comes next.
